Create-Cluster.md

```diff
@@ -79,11 +103,26 @@ Steps:
           kuberos-b7d5f755d-l8jb5   1/1       Running   0          1m
         ```
 1. Add WebOps group as admins
+
     ```
      $ kubectl apply -f ../../../cluster-config/rbac/webops-cluster-admin.yml
      clusterrolebinding.rbac.authorization.k8s.io "webops-cluster-admin" created
     ```
 
+### Recover the laa-fee-calculator application
 
+1. In the [cloud-platform-environments](https://github.com/ministryofjustice/cloud-platform-environments/tree/master/namespaces/cloud-platform-live-0.k8s.integration.dsd.io) repo, apply the laa-fee-calculator directories
 
-
+    ```
+    $ kubectl apply -f laa-fee-calculator-staging
+    namespace "laa-fee-calculator-staging" created
+    rolebinding.rbac.authorization.k8s.io "laa-fee-calculator-staging-admin" created
+    serviceaccount "circleci" created
+    role.rbac.authorization.k8s.io "circleci" created
+    rolebinding.rbac.authorization.k8s.io "circleci" created
+    ```
+1. In the [laa-fee-calculator](https://github.com/ministryofjustice/laa-fee-calculator) repo, deploy the laa-fee-calculator application.
+    1. Switch branch to `dd_dd_sqlite_and_k8s`
```

Comment:
True, but this will eventually merge into master. Should we change it to read `master` and add a note that it's currently pending a merge from that branch?